src/utils/errors/errorFormatter.js

```diff
@@ -13,14 +13,29 @@ export default function formatError (error) {
     return {
       message: 'Network error. Please check your internet connection.',
       code: 'NETWORK_ERROR',
-      details: {}
+      details: {
+        originalErrorMessage: error.message,
+        originalStack: error.stack
+      }
```

Comment:
Important: to be consistent, let's do that modification to all errors, not only the URL parsing failure one.
```suggestion
      details: error.message,
      stack: error.stack
```